packages/visualisation/src/components/ExperimentSection/index.js

```diff
@@ -104,11 +104,23 @@ const CheckItem = ({ text, setLayer, checked, color, borderOnly = false }) => {
 }
 
 const ExperimentSection = ({ activeLayers, currentParameters }) => {
+  const experimentId = currentParameters.id
+  const kibanaDashboardId = '767e6ae0-3ac8-11ed-bade-51cc9f3c8210'
```

Comment:
I think I landed in that this should just be part of the url since the panel-uuids in the query-params are probably linked to the dashboard-uuid.

Reply:
Ahh good point, I will keep it simple.